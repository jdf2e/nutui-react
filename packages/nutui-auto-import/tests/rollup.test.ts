import path from 'node:path'
import { rollupBuild, testFixtures } from '@sxzz/test-utils'
import { describe } from 'vitest'
import UnpluginStarter from '../src/rollup'

describe('rollup', async () => {
  const { dirname } = import.meta
  await testFixtures(
    '*.js',
    async (args, id) => {
      const { snapshot } = await rollupBuild(id, [
        UnpluginStarter({
          libraryName: '@nutui/nutui-react',
          style: 'css',
          taro: false,
        }),
      ])
      return snapshot
    },
    { cwd: path.resolve(dirname, 'fixtures'), promise: true }
  )
})
