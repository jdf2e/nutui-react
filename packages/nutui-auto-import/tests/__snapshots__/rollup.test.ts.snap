// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`rollup > basic.js 1`] = `
"// basic.js
// unplugin-starter injected
const num = 10;

export { num };
"
`;
